talk/morelanguage/stl.tex

```diff
@@ -97,27 +97,28 @@
     \item irrespective of
       \begin{itemize}
       \item the data handled
-      \item the container where data live
-      \item the iterator used to go through data
+      \item the container where the data live
+      \item the iterator used to go through data (almost)
       \end{itemize}
     \item examples
       \begin{itemize}
       \item for\_each, find, find\_if, count, count\_if, search
       \item copy, swap, transform, replace, fill, generate
       \item remove, remove\_if
-      \item unique, reverse, rotate, random, partition
-      \item sort, partial\_sort, merge, min, max
-      \item lexicographical\_compare, iota, accumulate, partial\_sum
+      \item unique, reverse, rotate, shuffle, partition
+      \item sort, partial\_sort, merge, make\_heap, min, max
+      \item lexicographical\_compare, iota, reduce, partial\_sum
```

Comment:
`random` does not exist and you should prefer `reduce` over `accumulate` if your `T` and operation are commutative.